src/components/TopicCard.jsx

```diff
@@ -0,0 +1,26 @@
+import "../App.css";
```

Comment:
As the UI is the same for Topic card as your other review cards, I would look to reuse this component. The same for your TopicArticleList. We can reuse rather than create duplicates.